### ISSUE TITLE
Running with tester gives values, but doesn't in osu!

### ISSUE THREAD

**Issue by icecapped** [apsun/AbsoluteTouchEx#5]
Exactly what the title says. I'm running an ELAN touchpad, which windows says is a precision touchpad in the settings. The tester log gives coordinate values, but it doesn't when launched with osu. Cursor also doesn't move when absolute mode is enabled in osu.

**Comment by apsun** [comment]
Did you enable raw input in osu?

**Comment by icecapped** [comment]
Yep.

**Comment by Jpe230** [comment]
Can confirm this to a certain degree.

It seems that your solution "works" since osu is responding when touching the touchpad when Absolute mode is turned on. 

Example 1:
-If I touch it, osu wants to exit. No matter where I click it wants to exit.

Example 2:
-If I'm in the song selection and I touch in an upper or lower area of the touchpad, the songs start to scroll, although the cursor never moves.

Examples 3:
-If I don't give any input to my touchpad, it starts to scroll up and down but the cursor remains static.

I tried running Osu in Fullscreen and Borderless with no luck. Also raw input is activated.

**Comment by apsun** [comment]
Sorry, I totally forgot about this thread..

>If I touch it, osu wants to exit. No matter where I click it wants to exit.

>If I don't give any input to my touchpad, it starts to scroll up and down but the cursor remains static.

Can you elaborate on this? What mode are you in when this occurs?

Do you have some touchpad gestures enabled? That's my best guess as to what's going on.

**Comment by Jpe230** [comment]
> Can you elaborate on this? What mode are you in when this occurs?
>>If I touch it, osu wants to exit. No matter where I click it wants to exit.

When absolute mode is both enabled and disabled, and I'm in the title screen and I want to move around (i.e. touching the touchpad), osu asks me if I want to exit from the game.

>>If I don't give any input to my touchpad, it starts to scroll up and down but the cursor remains static.

If I open the song selection and enable absolute mode, without any input (i.e. not touching the touchpad) it starts to scroll my songs up and down.

In all 3 cases the cursor in Osu! remains static, and can click buttons and select a song in Osu! if Absolute mode is turned on, but the cursor doesn't move at all making the game unplayable.

> Do you have some touchpad gestures enabled? That's my best guess as to what's going on.

Yes, only one, drag to scroll.

**Comment by icecapped** [comment]
Yeah I have the exact same stuff happen in the same examples @Jpe230

**Comment by apsun** [comment]
What touchpad are you using?

**Comment by Jpe230** [comment]
Apparently both of us are using Elan

**Comment by danil1petrov** [comment]
any solution? atloader osu.exe just starts the game without outputting to the cmd. only launching without a path to the game gives output.
